### ISSUE TITLE
testing at an arduino nano

### ISSUE THREAD

**Issue by maurimerel** [Edzelf/HDMI-simulator#1]
hi there , sorry to bother you , I`m trying to use your sketch with an arduino nano and I can`t get it working . 
I saw at the ports declarations for i2c comm are  3 and 4 . I`m really new at this so i was wondering if it`s not 4 and 5 pins .
Anyways y tested both options with same result ,  when I connect it to my laptop i should see a new monitor detected i guess but no.
Can you help me a little? thanks so much!!!!

**Comment by Edzelf** [comment]
It is a very old project.  I will check it, give me some time and I report back to you.

**Comment by Edzelf** [comment]
I used an Arduino -pro-mini like this:
![image](https://github.com/Edzelf/HDMI-simulator/assets/18257026/8143ded9-0831-4fb1-a2d5-179c65fb17d1)
The pins used are 3, 4 and 5.  Pins 3 (scl) and 4 (sda) are both pulled up by resistors of 4.7 k.  Maybe you have forgotten to add these.
I also included a c++ version of the code.  I'm not sure if this has been tested.

**Comment by maurimerel** [comment]
Ok , i`ll double check but I`m confused because wherever I check arduino pinouts I find references like this .
![arduino pro mini](https://github.com/Edzelf/HDMI-simulator/assets/78920768/e614a4b9-cee1-43b6-94ca-2666cdb2dff8)

A4 and A5 are the standard I2C pins but i guess you are using D3 D4 and D5 . Is that ok?? that I didn`t test.

**Comment by Edzelf** [comment]
Yes, that's right pin 5 is the "hotplug"-pin.  The I2C hardware is not used.  It is software bit-banging.
